docs/config/distillery.md

```diff
@@ -1,7 +1,7 @@
 # Release Configuration
 
 Looking for how to handle configuration of your app when using releases?
-Check out [Runtime Configuration](https://hexdocs.pm/distillery/runtime-configuration.html).
+Check out [Runtime Configuration](https://hexdocs.pm/distillery/config/runtime.html).
```

Comment:
Can you change this link to `[Runtime Configuration](runtime.md)`? The doc builder will sort out the actual live link - looks like this one got missed when things were rewritten. Thanks for the PR!